cmd/dbaas_list.go

```diff
@@ -56,12 +56,13 @@ func (c *dbaasServiceListCmd) cmdRun(_ *cobra.Command, _ []string) error {
 
 	out := make(dbaasServiceListOutput, 0)
 	res := make(chan dbaasServiceListItemOutput)
-	defer close(res)
+	done := make(chan struct{})
```

Comment:
The `done` chan is never closed. 
Is there any good reason not do it ?

Reply:
It is OK to leave a channel open. They are garbage collected if no longer in use.
Closing res is required as it's a condition to exit the loop L62.

Reply:
List command process will exit fast (after printing the results) so channel will not live long anyway and closing channel does not have any impact on the workflow. It would be correct to close the channel, but it this particular case I'd rather have one less line in the code.